src/lib/dlt_client.c

```diff
@@ -153,16 +154,17 @@ DltReturnValue dlt_client_init(DltClient *client, int verbose)
         dlt_vlog(LOG_INFO,
                  "Init dlt client struct with default port: %hu.\n",
                  servPort);
-
     return dlt_client_init_port(client, servPort, verbose);
 }
 
 DltReturnValue dlt_client_connect(DltClient *client, int verbose)
 {
+    const int yes = 1;
```

Comment:
As this value is used only if DLT_CLIENT_MODE_UDP_MULTICAST, I propose to define this line inside there. Same for line 167

Reply:
I don't agree with this comment.
Defining variables after labels (DLT_CLIENT_MODE_UDP_MULTICAST) is not valid in C. Although workarounds exists, in my opinion they are kinda ugly :)